cg/models/run_devices/illumina_run_directory_data.py

```diff
@@ -97,6 +94,25 @@ def sample_sheet_path(self) -> Path:
             self.get_sequencing_runs_dir(), DemultiplexingDirsAndFiles.SAMPLE_SHEET_FILE_NAME
         )
 
+    @property
+    def get_sequencing_completed_path(self) -> Path:
+        """Return the path to the sequencing completed file."""
+        sequencing_completed_paths: dict = {
```

Comment:
I guess at this point we just have to accept that Illumina changes their way of things over time... anyway hiseq x seem to have a SEQUENCING completed at and NOVASEQ 6000 a SEQUENCE completed at.

Reply:
🤦‍♂️